Jenkinsfile_CNP

```diff
@@ -23,27 +23,26 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 
 
 def vaultOverrides = [
-        'preview' : 'aat',
-        'spreview': 'saat'
+    'preview' : 'aat',
+    'spreview': 'saat'
 ]
 
 withPipeline(type, product, component) {
 
-    /*after('smoketest:aat') {
-        sh "rm -rf infrastructure/"
-    }*/
+
 
     env.S2S_URL_TO_TEST = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
     env.IDAM_URL_TO_TEST = "https://idam-api.aat.platform.hmcts.net"
-    env.TEST_URL = "http://rd-user-profile-api-aat.service.core-compute-aat.internal"
 
     overrideVaultEnvironments(vaultOverrides)
     loadVaultSecrets(secrets)
 
     enableDbMigration('rd')
     installCharts()
+    
     // **This is for demo. Uncomment below for AAT/AKS deployments**
```

Comment:
you can remove this comment since is not valid now since for env we require disableLegacyDeployment()

Reply:
Removed comment.